samples/customer-registry-quickstart/README.md

```diff
@@ -0,0 +1,89 @@
+It also contains the configuration to start a local Google Pub/Sub emulator that the Akka Serverless proxy will connect to.
+To start the proxy, run the following command from this directory:
+
+
+```
+docker-compose up
+```
+
+
+> On Linux this requires Docker 20.10 or later (https://github.com/moby/moby/pull/40007),
+> or for a `USER_FUNCTION_HOST` environment variable to be set manually.
+
+```
+docker-compose -f docker-compose.yml -f docker-compose.linux.yml up
+```
+
+To start the application locally, the `exec-maven-plugin` is used. Use the following command:
+
+```
+mvn compile exec:java
+```
+
+With both the proxy and your application running, any defined endpoints should be available at `http://localhost:9000`. In addition to the defined gRPC interface, each method has a corresponding HTTP endpoint. Unless configured otherwise (see [Transcoding HTTP](https://developer.lightbend.com/docs/akka-serverless/java/proto.html#_transcoding_http)), this endpoint accepts POST requests at the path `/[package].[entity name]/[method]`. For example, using `curl`:
+
+
+* Create a customer with:
+  ```
+  grpcurl --plaintext -d '{"customer_id": "wip", "email": "wip@example.com", "name": "Very Important", "address": {"street": "Road 1", "city": "The Capital"}}' localhost:9000  customer.api.CustomerService/Create
+  ```
+* Retrieve the customer:
+  ```
+  grpcurl --plaintext -d '{"customer_id": "wip"}' localhost:9000  customer.api.CustomerService/GetCustomer
+  ```
+* Query by name:
+  ```
+  grpcurl --plaintext -d '{"customer_name": "Very Important"}' localhost:9000 customer.view.CustomerByName/GetCustomers
+  ```
+* Change name:
+  ```
+  grpcurl --plaintext -d '{"customer_id": "wip", "new_name": "Most Important"}' localhost:9000 customer.api.CustomerService/ChangeName
+  ```
+* Change address:
+  ```
+  grpcurl --plaintext -d '{"customer_id": "wip", "new_address": {"street": "Street 1", "city": "The City"}}' localhost:9000 customer.api.CustomerService/ChangeAddress
+  ```
+
+## Deploying
+
+To deploy your service, install the `akkasls` CLI as documented in
+[Setting up a local development environment](https://developer.lightbend.com/docs/akka-serverless/getting-started/set-up-development-env.html)
+and configure a Docker Registry to upload your docker image to.
+
+You will need to update the `akkasls.dockerImage` property in the `pom.xml` and refer to
+[Configuring registries](https://developer.lightbend.com/docs/akka-serverless/deploying/registries.html)
+for more information on how to make your docker image available to Akka Serverless.
+
+Finally you can or use the [Akka Serverless Console](https://console.akkaserverless.com)
+to create a project and then deploy your service into the project either by using `mvn deploy`,
+through the `akkasls` CLI or via the web interface. When using `mvn deploy`, Maven will also
```

Comment:
The way this sentence reads, it sounds like they can use mvn deploy in the web interface. It needs to be rewritten because if they use the console, they still need to package and publish the image. Is there a separate mvn target for that that doesn't deploy? If not, we should just tell them to use the CLI.

Reply:
that is a great question that I don't know the answer to offhand. let's ticketify